### PR TITLE
Add documentation bring your own kafka

### PR DESCRIPTION
**Pull request by JansenNick**

### CODE REVIEW

**Comment by johannahemminger** [comment]
Thanks @JansenNick for this PR! When should this be merged? ASAP, or on 10GA release date?

**Comment by JansenNick** [comment]
@johannahemminger once we release the 3.6.0 business events module with the bring your own kafka changes, Pritika and some internal teams are still testing the new module.

With regards to the updated documentation, feel free to make edits as you see fit.

File: content/en/docs/appstore/modules/business-events.md [hunk 1]
```diff
@@ -348,9 +348,7 @@ Business Events offers three different deployment models:
 1. Deploy locally with the [Local Setup Tool](https://github.com/mendix/event-broker-tools)
 2. Free apps use a free multi-tenant Event Broker
 3. Production apps use the [Mendix Event Broker](#mendix-event-broker) running in the Mendix Cloud
-
-{{% alert color="info" %}}
-Currently, business events can only be deployed to the [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/), with other deployment models expected in forthcoming releases.{{% /alert %}} 
+4. Bring your own Kafka
```
Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: Here we want to please say (Beta) behind it.  .... 

_Mendix apps running in Mendix for Private Cloud (Bring your own Kafka).  This functionality is still in beta testing_

File: content/en/docs/appstore/modules/business-events.md [hunk 2]
```diff
@@ -372,11 +370,17 @@ You can also use your own event broker cluster.
 
 #### 6.3.1 Warning Message When Deploying
 
-If you deploying an app to production that contains a published business event service, you might receive a warning that it is not possible to enable the event broker service. If you see this message, do the following in the [Services](/developerportal/deploy/environments/#services) tab of the production environment screen:
+If you are deploying an app to production that contains a published business event service, you might receive a warning that it is not possible to enable the event broker service. If you see this message, do the following in the [Services](/developerportal/deploy/environments/#services) tab of the production environment screen:
```
Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: "you may receive a warning" ?

File: content/en/docs/appstore/modules/business-events.md [hunk 3]
```diff
@@ -372,11 +370,17 @@ You can also use your own event broker cluster.
...
 
 1. Select the **Enable** checkbox for the environment.
 2. Transport the mda to an environment.
 3. Restart the environment.
+   
+### 6.4 Bring your own Kafka
+
+To deploy to a private cloud using your own Kafka cluster, you can make use of the the `EventBrokerSpace` constant. This constant can be set to values like test, acceptance or production based on in which environment (or say namespace in Kubernetes) your Mendix app is running. Setting this constant would mean that each Business Event would have its own Kafka topic for environment. The topic would be of the form businessevents.<channel>.<EventBrokerSpace>.
```
Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: Here we need to add that BYOK is in a beta stage (under licensing we should add more info on this)

Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: "Kafka topic for environment" does not read well

Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: Do we explain channel anywhere else?

File: content/en/docs/appstore/modules/business-events.md [hunk 4]
```diff
@@ -372,11 +370,17 @@ You can also use your own event broker cluster.
...
 #### 6.3.1 Warning Message When Deploying
 
-If you deploying an app to production that contains a published business event service, you might receive a warning that it is not possible to enable the event broker service. If you see this message, do the following in the [Services](/developerportal/deploy/environments/#services) tab of the production environment screen:
+If you are deploying an app to production that contains a published business event service, you might receive a warning that it is not possible to enable the event broker service. If you see this message, do the following in the [Services](/developerportal/deploy/environments/#services) tab of the production environment screen:
 
 1. Select the **Enable** checkbox for the environment.
 2. Transport the mda to an environment.
 3. Restart the environment.
+   
+### 6.4 Bring your own Kafka
```
Comment (stefanus-heath) [inline content/en/docs/appstore/modules/business-events.md]: _Mendix for Private Cloud (Bring your own Kafka)_

**Review by stefanus-heath — CHANGES_REQUESTED** [review]

File: content/en/docs/appstore/modules/business-events.md (hunk 3 above)
Comment (johannahemminger) [reply content/en/docs/appstore/modules/business-events.md]: Yes I'll link to it!

**Pull request merged**